src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx

```diff
@@ -116,16 +96,18 @@ export const CreateQuizSecondCard: React.FC = observer(() => {
     onSubmit: (data) => {
       if (Object.keys(formik.errors).length === 0) {
         if (QuizStore.selectedQuestionID) {
-          QuizStore.editQuestion(QuizStore.selectedQuestionID, {
-            ...formik.values,
-            answerList,
-          });
-          handleFirebaseAdd();
-        } else {
-          QuizStore.addQuestion(
-            { ...formik.values, answerList },
-            correctResponseIndexRadio
-          );
+          if (QuizStore.selectedQuestionIsEmpty) {
+            QuizStore.editQuestion(QuizStore.selectedQuestionID, {
+              ...formik.values,
+              answerList,
+            });
+            QuizStore.addEmptyQuestion();
+          } else {
+            QuizStore.editQuestion(QuizStore.selectedQuestionID, {
+              ...formik.values,
+              answerList,
+            });
```

Comment:
The function `editQuestion` is called exactly the same for `if` and for `else`. To simplify, it could be placed right before the `if`.

Reply:
I have a problem here. I tried what you said before the pull request, but it's not working the same, strangely.